cpuid/cpuid.cpp

```diff
@@ -85,12 +87,13 @@ int main()
        if ( CPUInfo[2] & 0x10000000 ) // bit 28
           printf("AVX\n");
 
+#ifndef __MINGW32__
        if ( CPUInfo[2] & 0x8000000 ) // bit 27
```

Comment:
Why are you not checking or OXSAVE?

Reply:
The check for that was disabled for some reason causing an unused variable warning.

Reply:
Did you have to remove `_xgetbv` for MinGW which was the codepath to test osxsave?

Reply:
I found it `if 0`'ed out.  If I remove the `if 0`, I get the following error:

```
In file included from C:/msys64/mingw64/lib/gcc/x86_64-w64-mingw32/13.2.0/include/x86gprintrin.h:99,
                 from C:/msys64/mingw64/lib/gcc/x86_64-w64-mingw32/13.2.0/include/x86intrin.h:27,
                 from C:/msys64/home/jpmugaas/directxmath-PKGBUILD/src/directxmath-dec2022/Tests/cpuid/cpuid.cpp:17:
C:/msys64/mingw64/lib/gcc/x86_64-w64-mingw32/13.2.0/include/xsaveintrin.h: In function 'int main()':
C:/msys64/mingw64/lib/gcc/x86_64-w64-mingw32/13.2.0/include/xsaveintrin.h:60:1: error: inlining failed in call to 'always_inline' 'long long int _xgetbv(unsigned int)': target specific option mismatch
   60 | _xgetbv (unsigned int __A)
      | ^~~~~~~
C:/msys64/home/jpmugaas/directxmath-PKGBUILD/src/directxmath-dec2022/Tests/cpuid/cpuid.cpp:325:45: note: called from here
  325 |         xcr0 = static_cast<uint32_t>(_xgetbv(0));

```

Reply:
Fixed.